doc/modules/cross_validation.rst

```diff
@@ -524,11 +524,30 @@ Cross-validation iterators with stratification based on class labels
 --------------------------------------------------------------------
 
 Some classification problems can exhibit a large imbalance in the distribution
```

Comment:
```suggestion
Some classification problems can exhibit very rare classes.
```
I'd like to get rid of the term "imbalance". Imbalance is not a problem.

Reply:
I agree, we could also just rename "classification problems" to "classification tasks".